2024-04-18 05:22:53 ERROR Elements not present after 0.5 seconds on page: https://capital.com/cn/trade-etfs
2024-04-18 05:22:53 ERROR 
Traceback (most recent call last):
  File "/home/runner/work/TestCapitalComPySe/TestCapitalComPySe/pages/base_page.py", line 85, in inner_function
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/work/TestCapitalComPySe/TestCapitalComPySe/pages/base_page.py", line 304, in element_is_visible
    return Wait(self.driver, timeout).until(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/.cache/pypoetry/virtualenvs/testcapitalcompyse-CcJ6hTD8-py3.12/lib/python3.12/site-packages/selenium/webdriver/support/wait.py", line 105, in until
    raise TimeoutException(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: 
Stacktrace:
#0 0x55ad33c5d863 <unknown>
#1 0x55ad339538c6 <unknown>
#2 0x55ad3399e618 <unknown>
#3 0x55ad3399e6d1 <unknown>
#4 0x55ad339e1744 <unknown>
#5 0x55ad339c05cd <unknown>
#6 0x55ad339dec19 <unknown>
#7 0x55ad339c0343 <unknown>
#8 0x55ad33991593 <unknown>
#9 0x55ad33991f5e <unknown>
#10 0x55ad33c2184b <unknown>
#11 0x55ad33c257a5 <unknown>
#12 0x55ad33c0f571 <unknown>
#13 0x55ad33c26332 <unknown>
#14 0x55ad33bf487f <unknown>
#15 0x55ad33c4c728 <unknown>
#16 0x55ad33c4c8fb <unknown>
#17 0x55ad33c5c9b4 <unknown>
#18 0x7fe253e94ac3 <unknown>